### ISSUE TITLE
Theme font colors

### ISSUE THREAD

**Issue by fergardi** [marcosmoura/vue-material#352]
Hello!

I'm following the detailed coloring specification at https://material.io/guidelines/style/color.html#color-color-palette and found a little bug. For example, if I choose the TEAL color code, the main font-color (used in the toolbar text and primary icons) is white as specified, but if I choose the LIGHT-GREEN with 700 hue, the default value is black instead to the specification. It will be easy to correct using CSS but wanted to share anyway.

TL;DR: font color mismatching when using google color palette hue.

Thank you for your great work.

**Comment by marcosmoura** [comment]
I tried your example on [Codepen](http://codepen.io/vue-material/pen/WGavBE) and the colors are white. I coundn't reproduce this bug.

**Comment by fergardi** [comment]
You are completely right. I misunderstood the specification.

Thank you for your time.
